### ISSUE TITLE
Search is broken on readthedocs

### ISSUE THREAD

**Issue by cyberw** [locustio/locust#1391]
Clicking search does nothing at all. I want to fix it but dont have the time right now :-/

Probably related to the sphinx-search feature.

**Comment by heyman** [comment]
Hm, interesting. I can't reproduce the error when building the docs locally, which makes it harder to debug.

**Comment by cyberw** [comment]
That didnt help...

**Comment by cyberw** [comment]
@heyman It started working when I commented out this code in conf.py. But I'm guessing you put it there for a reason? :)

```
 # Custom CSS overrides
 html_static_path = ["_static"]
 html_context = {
     "css_files": ["_static/theme-overrides.css"],
 }
```

**Comment by heyman** [comment]
Yes, it adds some custom CSS which makes it possible to have table cells with multiple lines in the ReadTheDocs theme.

**Comment by heyman** [comment]
Now fixed in e91810596bd621f149b75f0f99e78475bfaf8780!
